docs/yaml.md

```diff
@@ -10,6 +10,11 @@ The yaml configuration specifies the components used to build up an image . All
 are downloaded at build time to create an image. The image is self-contained and immutable,
 so it can be tested reliably for continuous delivery.
 
+Components are specified as Docker images which are pulled from registry during build if they
```

Comment:
"from a registry" (or "the", but I think not)

Reply:
tahnsk. pushed a fix